rllib/tuned_examples/a3c/cartpole-a2c.yaml

```diff
@@ -1,9 +1,10 @@
-cartpole-a2c-torch:
+cartpole-a2c:
     env: CartPole-v0
     run: A2C
     stop:
         episode_reward_mean: 100
```

Comment:
Does it actually make sense to move these all into the general set of tuned examples? The problem is that the stop conditions for the regression tests are set quite conservatively.

Reply:
Well, they are certainly "tuned" (or should be), so I think that's fine. I agree that we should up the stop-condition to make things harder and more solid. 100 for CartPole is definitely not enough. Let's make all these 150. I think the Pendulum tests are ok.